cmd/commands/server/command_server.go

```diff
@@ -112,21 +112,7 @@ func (cmd *Command) Start() (err error) {
 		}
 	}
 
-	go func() {
-		for {
-			select {
-			case <-time.After(1 * time.Minute):
-				err := cmd.mysteriumClient.PingProposal(proposal, signer)
-				if err != nil {
-					log.Error("Failed to ping proposal", err)
-					// do not stop server on missing ping to discovery. More on this in MYST-362 and MYST-370
-				}
-			case <-stopPinger:
-				log.Info("Stopping proposal pinger")
-				return
-			}
-		}
-	}()
+	go pingProposalLoop(proposal, cmd.mysteriumClient, signer, stopPinger)
```

Comment:
Code is too fat here. Would be nice to move to something like `discoveryAnnouncementLoop()`